apps/full-site-editing/full-site-editing-plugin/full-site-editing/blocks/site-credit/index.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable import/no-extraneous-dependencies */
+/**
+ * WordPress dependencies
+ */
+import { registerBlockType } from '@wordpress/blocks';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import edit from './edit';
+import './style.scss';
+
+registerBlockType( 'a8c/site-credit', {
+	title: __( 'WordPress.com Credit' ),
+	description: __( "This block tells the wolrd that you're using WordPress.com." ),
+	icon: 'wordpress-alt',
+	category: 'layout',
+	supports: {
+		align: [ 'right', 'center', 'left' ],
```

Comment:
I assume that we'll also have block alignment options (e.g. wide and full) down the line.
For the other FSE blocks I chose to use `align` for block alignment, and `textAlign` for text alignment.

Would you mind keeping it consistent here as well? 🙇

Reply:
@Copons I'll look into this. I wasn't sure which made the most sense since the reality is that we're aligning several spans in a flex container, but maybe I can get around that somehow...

Reply:
I've changed this to `align: [ 'full', 'wide' ]` and now we can set text alignment to left/right/center. This seems to be working very well :)

I didn't keep the alignment options above because they don't seem to play well with text align _and_ it seems to mostly duplicate what text align already does. (It'd probably be weird for a user to have to set "align center" and "text align center" in order to center align it. This way, they only have to set "text align center")

Reply:
Yup, this is exactly what we did for the other FSE blocks!  🙂